br/pkg/lightning/backend/tidb/tidb.go

```diff
@@ -883,8 +883,7 @@ func FetchTableAutoIDInfos(ctx context.Context, exec utils.QueryExecutor, tableN
 	var autoIDInfos []*TableAutoIDInfo
 	for rows.Next() {
 		var (
-			dbName, tblName, columnName, idType string
-			nextID                              int64
```

Comment:
can change the type of `nextID` to uint64, so SQL drvier will do the conversion work.

Reply:
modified